app/vep/models/upload_vcf_files.py

```diff
@@ -75,6 +76,11 @@ async def stream(self):
             async for chunk in self.request.stream():
                 body_validator(chunk)
                 self.parser.data_received(chunk)
+
+            if self.filename == 'temp_name':
+                os.rename(self.filepath, os.path.join(self.temp_dir, self._input_file.multipart_filename))
+                self.filename = self._input_file.multipart_filename
+                self.filepath = os.path.join(self.temp_dir, self._input_file.multipart_filename)
```

Comment:
these three lines are repeating the long and non-obvious incantation `self._input_file.multipart_filename`. Don't you want to assign it to a readable variable and then use that?

```
filename = self._input_file.multipart_filename # read the file name from the multipart form data
self.filename = filename
self.filepath = os.path.join(self.temp_dir, filename)
```

Reply:
I'm trying to avoid unnecessary variables that will only serve as a temp data holder. 
But self.filepath can be simplified.
`self.filepath = os.path.join(self.temp_dir, self.filename)`